profile_library/eq-3/HmIP-DRSI1/model.json

```diff
@@ -1,18 +1,23 @@
 {
-  "measure_description": "Manually measured",
-  "measure_method": "manual",
-  "measure_device": "From manufacturer specifications",
+  "measure_description": "Manually measured with dummy load attached at 7.7W (average over 300s) multiple times, than calculated average of that.",
+  "measure_device": "Shelly PM Mini Gen3 (S3PM-001PCEU16)",
+  "measure_method": "script",
+  "measure_settings": {
+    "VERSION": "v1.17.1:docker"
+  },
   "name": "HmIP-DRSI1",
-  "standby_power": 0.2,
-  "standby_power_on": 0.9,
+  "standby_power": 0.25,
+  "standby_power_on": 0.3,
   "sensor_config": {
     "power_sensor_naming": "{} Device Power",
     "energy_sensor_naming": "{} Device Energy"
   },
   "device_type": "smart_switch",
   "calculation_strategy": "fixed",
-  "only_self_usage": true,
-  "created_at": "2024-09-06T18:37:40",
+  "fixed_config": {
```

Comment:
why did you remove only_self_usage here?
That is the prefered approach. We don't want fixed 0 added here.
Please keep the configuration conforming the example configuration as documented here: https://docs.powercalc.nl/library/device-types/smart-switch/

Reply:
This device does not have an integrated power meter.

Reply:
When the device does not have an integrated power meter `only_self_usage` needs to be false (or not provided), but `fixed_config` needs to be removed.
In this scenario the user will be provided with a dialog where he can put the fixed value for connected appliance.
When you set `fixed_config` to 0 this will not work.

So the configuration needs to be as in the docs:

```json
{
  "standby_power": 0.25,
  "standby_power_on": 0.3,
  "device_type": "smart_switch",
  "calculation_strategy": "fixed"
}
```

or

```json
{
  "standby_power": 0.25,
  "standby_power_on": 0.3,
  "device_type": "smart_switch",
  "calculation_strategy": "fixed",
  "only_self_usage": false
}
```

Reply:
I see. I was not aware that providing a fixed value will disable the option.